### ISSUE TITLE
[CKV_SECRET_6] Check produces false positive based on variable name

### ISSUE THREAD

**Issue by srgoni** [bridgecrewio/checkov#5469]
**Describe the issue**
Checkov is too eager when estimating entropy on the CKV_SECRET_6 check, leading to false positives.

**Examples**
A very basic Terraform example:
```hcl
locals {
  sm_name_secrets = "my-custom-resource-name"
}
```
leads to:
```
Check: CKV_SECRET_6: "Base64 High Entropy String"
        FAILED for resource: ....
 ```

**Version (please complete the following information):**
2.4.5

**Additional context**
The check shouldn't complain about a high-entropy string simply based the variable name - especially when it doesn't actually contain a lot of entropy and isn't used as a secret value anywhere.

Possible duplicates: #5405 #4604

**Comment by tsmithv11** [comment]
Thanks for reaching out! Closing as a duplicate of #5405
